frontend/src/components/feed/Feed.js

```diff
@@ -30,8 +34,10 @@ const Feed = ({ navigate }) => {
     if(token) {
       return(
         <>
-          <h2>Posts</h2>
-            <button onClick={logout}>
+          <h2 className='h2'>Posts</h2>
+            <button className='btn' onClick={() => setForm(!form) }>Add Post</button>
```

Comment:
Could we potentially look at renaming the className on line 38 to be more clear about what the 'btn' does?